packages/datetime-picker/src/web.ts

```diff
@@ -11,6 +11,6 @@ export class DatetimePickerWeb
   implements DatetimePickerPlugin
 {
   public async present(_options: PresentOptions): Promise<PresentResult> {
-    throw this.unimplemented('Not implemented on web.');
+    throw this.unimplemented('Not implemented on web. Options: ' + _options);
```

Comment:
Please revert this change.

Reply:
Okay, i just tried to fix the linter errors^^